repo/db/chat_test.go

```diff
@@ -1,27 +1,41 @@
-package db
+package db_test
 
 import (
-	"database/sql"
-	"github.com/OpenBazaar/openbazaar-go/repo"
 	"sync"
 	"testing"
 	"time"
-)
 
-var chdb repo.ChatStore
-
-func init() {
-	setupDB()
-}
+	"github.com/OpenBazaar/openbazaar-go/repo"
+	"github.com/OpenBazaar/openbazaar-go/repo/db"
+	"github.com/OpenBazaar/openbazaar-go/schema"
+)
 
-func setupDB() {
-	conn, _ := sql.Open("sqlite3", ":memory:")
-	initDatabaseTables(conn, "")
-	chdb = NewChatStore(conn, new(sync.Mutex))
+func buildNewChatStore() (repo.ChatStore, func(), error) {
+	appSchema := schema.MustNewCustomSchemaManager(schema.SchemaContext{
```

Comment:
I made an attempt, but realized it would expose the private `openbazaarSchemaManager` which I was trying to avoid. 

Nothing would be stopping a dev from misusing other parts of the language so maybe applying the same diligence around our Must usage would be preferred? Rather than exposing private internals and the dev accidentally breaking internal state of the manager?